timeline.tex

```diff
@@ -1,24 +1,48 @@
 \section{Roadmap and Timeline} \label{sec:timeline}
 
-\TODO{Phil}{Replace with milestones figure from DPRPT. Explain uncertainty. What happens if commissioning gets squeezed? Ops does SV, as explained in intro.tex.} 
-
-This timeline provides a list of key dates  related to the \es program.
+The timeline shown in Table~\ref{tab:milestones} below provides a list of key dates related to the \es program.
 It will be updated with more dates related to the development of the \esp as they are defined.
 
-\textbf{2021-11-12:} Issue first version of the \ro Plans for an \esp.
-
-\textbf{2023-01-20:} Commissioning Camera (ComCam) on sky, engineering first light.
-
-\textbf{2023-07-01:} LSST Science Camera (LSSTCam) on sky.
-
-\textbf{2023-10-01:} Start of the \svs.
-
-\textbf{2023-12-18:} Current earliest completion date for construction.
-
-\textbf{2024-07-01:} Current latest completion date for construction.
-
-\textbf{2024-04-01:} Planned start of Operations Phase.
-
-\textbf{2024-04/09:} Current forecast date for Data Preview 2, LSSTCam On-Sky Data.
-
-\textbf{2025-02/07:} Current forecast date for Data Release 1.
+The date ranges in the Table are derived from the ``Celebratory Milestones'' list published at monthly intervals on the Rubin Project website;\footnote{\url{https://www.lsst.org/about/project-status}} shown here are the milestones as of October 2022, which use Project controls data from August 2022.
+
+Milestone dates are given as min-max ranges to indicate their uncertainty.
+Typically the near date corresponds to the current Project forecast, plus any additional operational uncertainty.
+The late date corresponds (approximately) to the current Project ``late date'' plus any additional operational uncertainty: this late date cannot be surpassed without the Project rebaselining its schedule.
+
+The late dates for the DP2 and DR1 data release milestones allow for the possibility that the Project completes within its late date, but reduces the amount of on-sky LSST Cam commissioning time in order to do so.
+In this eventuality, the operations team would spend up to 3 months at the start of the full/survey operations phase doing any remaining Science Validation Survey observations, such that DP2 could be realized as planned.
```

Comment:
phase **completing** any remaining Science Validation Survey observations, such that DP2 could be realized as planned.

Reply:
Good, thanks - adopting. BTW Did you know you can suggest changes directly, with that little page +/- icon? Makes GitHub more like Google docs, which is cool (-: